### PR TITLE
DevWs for www.easyiot.tech containing containing ZigBee Switch

### PR DESCRIPTION
**Pull request by jwg-123**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -31,6 +31,9 @@ metadata {
 		// eWeLink
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink Outlet", ocfDeviceType: "oic.d.smartplug" //eWeLink SmartPlug (SA-003)
 		fingerprint profileId: "0104", inClusters: "0000,0003,0004,00005,0006", outClusters: "0000", manufacturer: "eWeLink", model: "ZB-SW01", deviceJoinName: "eWeLink Switch"
+        
+        // LELLKI
+        fingerprint profileId: "0104", inClusters: "0000,0003,0004,00005,0006", outClusters: "0000", manufacturer: "LELLKI", model: "JZ-ZB-001", deviceJoinName: "LELLKI Switch"
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy:36]: Please fix indentation here (change 4 spaces to 1 tab) and add a Raw description in a comment after Your fingerprint.

Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please fix indentation here (change 4 spaces to 1 tab).

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -108,9 +111,6 @@ metadata {
 		fingerprint manufacturer: "Jasco Products", model: "43094", deviceJoinName: "Enbrighten Switch" //Enbrighten, Plug-in Smart Switch 43094, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43102", deviceJoinName: "Enbrighten Outlet", ocfDeviceType: "oic.d.smartplug" //Enbrighten, In-Wall Smart Outlet 43102, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
 		fingerprint manufacturer: "Jasco Products", model: "43076", deviceJoinName: "Enbrighten Switch" //Enbrighten, In-Wall Smart Switch 43076, Raw Description: 01 0104 0100 00 06 0000 0003 0004 0005 0006 0B05 02 000A 0019
-		
-		// Focalcrest
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", outClusters: "0019", manufacturer: "Focalcrest", model: "SRB01", deviceJoinName: "Focalcrest Switch" // In-Wall Relay Switch
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please add Your changes on a new branch (or rebase existing one)

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Comment by k-morawski** [comment]
Dear @jwg-123 , could you please close this pull request? I believe it is obsolete now as you requested a newer pull request here: https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/62113 
Many thanks!

**Comment by KKlimczukS** [comment]
Since #62113 was approved and  merged, I'm closing this PR

**Pull request closed without merging**